products/rtpengine.md

```diff
@@ -0,0 +1,40 @@
+---
+title: rtpengine 
+category: server-app
+permalink: /rtpengine
+versionCommand: rtpengine -v
+releaseDateColumn: true
+
+identifiers:
+-   repology: rtpengine
+
+releases:
+
+-   releaseCycle: "mr13.0"
+    releaseDate: 2024-10-01
+    eol: false
+    latest: "mr13.0.1"  
```

Comment:
Is there release notes documenting what changes between versions ?

Reply:
for this particular release : https://www.sipwise.com/releases/2024/mr13.0

Reply:
I added the link for every releases

Reply:
Mmmh, that will be hard to keep them up-to-date. Only documenting the major version (https://www.sipwise.com/releases/2024/mr13.0, https://www.sipwise.com/releases/2024/mr13.1...) would be better IMO.

Reply:
agree.